infra/main.bicep

```diff
@@ -138,18 +143,21 @@ var defaultDeploymentSettings = {
   name: environmentName
   principalId: principalId
   principalType: principalType
+  resourceToken: primaryResourceToken
   stage: environmentType
   tags: {
     'azd-env-name': environmentName
     'azd-env-type': environmentType
     'azd-owner-email': ownerEmail
     'azd-owner-name': ownerName
+    ResourceToken: primaryResourceToken
   }
   workloadTags: {
     WorkloadIdentifier: environmentName
     WorkloadName: environmentName
     Environment: environmentType
     OwnerName: ownerEmail
+    ResourceToken: primaryResourceToken
```

Comment:
Won't having this in tags be sufficient? Is there a scenario where we apply workloadTags but not tags?